### ISSUE TITLE
PCB Drawing

### ISSUE THREAD

**Issue by Tomi8609** [ElKaPa/opel-can-swc-adapter#1]
I would like to know if you have a PCB drawing for it?

On the wiring diagram I can't see where the steering wheel button output is

**Comment by ElKaPa** [comment]
Yes, I have created a PCB for this. Output for steering wheel remote to
radio is J3

Tomi8609 ***@***.***> schrieb am Mi., 18. Sept. 2024, 09:26:

**Comment by Tomi8609** [comment]
Thanks for the quick reply

Could you send or upload this pcb drawing?

**Comment by Tomi8609** [comment]
I guess you have no PCB drawing
???

**Comment by Tomi8609** [comment]
Hello

Can you help me with what I should change in the ino file to enter the secret menu by long pressing the two phone buttons on the left side? This works with the "Setting" button on the original radio and the "BC" button is used to step through the menu. 
- 201#01FF00 - Enter the BC menu
- 201#01C000 - Return to BC
